entities/mqtt/text/mtrxpi/audio_visualiser/high_magnitude_hex_color.yaml

```
@@ -0,0 +1,17 @@
+---
+  - text:
```

yamllint (1.35.1)

[error] wrong indentation: expected 0 but found 2

(indentation)